jobs/cronjob-aws-ocp-snap.yaml

```diff
@@ -0,0 +1,206 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: aws-ocp-snap
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    annotations:
+      description: Keeps track of changes in the application image
+    name: ${NAME}
+    labels:
+      template: aws-ocp-snap
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    annotations:
+      description: Defines how to build the application
+    name: ${NAME}
+    labels:
+      template: aws-ocp-snap
+  spec:
+    completionDeadlineSeconds: "1800"
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ${NAME}:latest
+    runPolicy: Serial
+    source:
+      git:
+        uri: https://github.com/redhat-cop/openshift-management.git
+      contextDir: /images/aws-ocp-snap
+    strategy:
+      dockerStrategy:
+        from:
+          kind: DockerImage
+          name: centos:7
+      type: Docker
+    triggers:
+    - type: ConfigChange
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: aws-secret
+    labels:
+      template: aws-ocp-snap
+  data:
```

Comment:
This can be changed to `stringData` which will allow the user to provide raw values instead of pre-calculated base64 values

Reply:
Done